ui/start_ui.py

```diff
@@ -1024,22 +1134,32 @@ def _init_ui(self):
                         label="Top k",
                         info="The number of highest probability vocabulary tokens to keep for top-k-filtering.",
                     )
-
                 with gr.Tab("Dialogue"):
                     chatbot = gr.Chatbot(
                         elem_id="chatbot",
                         label="chatbot",
                         elem_classes="disable_status",
                     )
 
+                    with gr.Row():
+                        with gr.Accordion("Image", open=False, visible=True):
+                            image = gr.Image(type="pil")
+                    with gr.Row():
+                        endpoint_value = "http://127.0.0.1:8000/v1/chat/completions"
+                        model_endpoint = gr.Text(
+                            label="Model Endpoint", value=endpoint_value, scale=1
+                        )
```

Comment:
Is it better for the default value to be None? We can set placeholder to let users know that it can be a pre-deployed endpoint or the endpoint returned from deployment module.